emission/tests/exportTests/TestExportPipelineBackgroundLocation.py

```diff
@@ -0,0 +1,39 @@
+from future import standard_library
+standard_library.install_aliases()
+from builtins import *
+import unittest
+import json
+import bson.json_util as bju
+import emission.storage.timeseries.abstract_timeseries as esta
+import gzip
+import emission.tests.common as etc
+import emission.pipeline.export_stage as epe
+
+class TestDataReadingSamples(unittest.TestCase):
+    def readDataFromFile(self, dataFile):
+        with open(dataFile) as dect:
+            raw_data = json.load(dect, object_hook = bju.object_hook)
+            return raw_data
+
+    def testReadDataFromFile(self):
+        raw_data = self.readDataFromFile("emission/tests/data/real_examples/shankari_2015-07-22")
+        self.printData(raw_data)
```

Comment:
My link to the PR on @corinne-hcr's branch is a good example of a commit message with "Testing done", but it is not actually a unit test. I just wanted to show her how to read data since she was struggling with it. As before, I would suggest looking at the pipeline tests or the user input tests for examples of real unit tests.

The testing that you are doing is on the right track, but you don't want to check the results manually. We want automated tests, so they will run on every commit + periodically and ensure that there are no regressions.
We currently have 218 automated tests, I am not signing up to view the results manually at this point.

https://github.com/e-mission/e-mission-server/runs/2947949822
```
web-server_1  | ----------------------------------------------------------------------
web-server_1  | Ran 218 tests in 559.069s
```

You would implement automated tests using `assertEqual` and similar.
https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertEqual (scroll up for full list)